packages/terra-date-time-picker/src/terra-dev-site/doc/date-time-picker/KeyboardShortcuts.3.doc.mdx

```diff
@@ -0,0 +1,28 @@
+import { Badge } from 'terra-date-time-picker/package.json?dev-site-package';
+
+<Badge />
+
+# Keyboard Shortcuts for Terra Date Time Picker
+
+The Terra Date Time Picker offers additional functionality with specific keyboard shortcuts to save time when interacting with date and time inputs in terra-date-time-picker.
+
+Pressing any of the supported keys while the input is in focus will populate the inputs with the listed shortcut response.
+
+| Key / Sequence | Input | Description                                                                                                                                                                                  |
+|----------------|-------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| T              | Date  | Sets the date value to the current date (Today).                                                                                                                                             |
+| -              | Date  | Decrements the day value by 1 day. If blank or not a valid date, input will set value to current date minus 1 day.                                                                           |
+| +              | Date  | Increments the day value by 1 day. If blank or not a valid time, input will set value to current date plus 1 day.                                                                            |
+| N              | Time  | Sets the time value to the current time (Now). Also sets date to the current date if there is no date or invalid date                                                                        |
+| -              | Time  | Decrements the minute value by 1 minute. If blank or not a valid time or date, input will set date to current date and value to current time minus 1 minute.                                 |
```

Comment:
@xenoworf I'll remind you that our requirements for these specific hotkey interactions are based on matching existing learned behaviors from internal win32 components. The T/N suite of hotkeys/keyboard shortcuts are not per-input node focused, but per date/time object focused, as such they only will ever affect a single node per concept (day for date, hour for time).

 E.G. pressing T sets today, we will add pressing M sets it to the first of the month, Y sets it to the first of the year. Plus (+) and minus (-) in this case are not synonymous with the up and down arrow in each input note unit, but work with the same consistency as the previous T, M, Y does, since the +/- are used as modifiers to those keys.